src/realm.json

```diff
@@ -2910,7 +2918,7 @@
             "alias": "CONFIGURE_TOTP",
             "name": "Configure OTP",
             "providerId": "CONFIGURE_TOTP",
-            "enabled": "${REALM_OTP_ENABLED:true}",
+            "enabled": "${OTP_ENABLED:true}",
```

Comment:
Did this still end up being a breaking change due to the removal of the `REALM_` prefix? I think that's okay since this only gets read in/evaluated at install time, just wanted to call it out and ask.

Reply:
Actually in thinking this through the change might be a bit more problematic, I dropped a comment on the core side - https://github.com/defenseunicorns/uds-core/pull/1102/files#r1884100380